From b629e068e3c6072a381e9348dbaacb8e4877dc65 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Wed, 23 Jun 2021 17:12:00 +0300
Subject: [PATCH 06/18] minigbm: Add const to the backend::name

... to avoid compile-time error on C++:

   error: ISO C++11 does not allow conversion from string
   literal to 'char *' [-Werror,-Wwritable-strings]

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
Change-Id: I176eb657f72e92d6b5c7c3b25c78c56f776c20ab
---
 drv_priv.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drv_priv.h b/drv_priv.h
index 5ec9fe7..fceffc0 100644
--- a/drv_priv.h
+++ b/drv_priv.h
@@ -75,7 +75,7 @@ struct driver {
 };
 
 struct backend {
-	char *name;
+	const char *name;
 	void (*preload)(bool load);
 	int (*init)(struct driver *drv);
 	void (*close)(struct driver *drv);
-- 
2.39.2

